stories/icon-button/props-table.tsx

```diff
@@ -6,17 +6,42 @@ const DATA: ApiTableSectionProps[] = [
     {
         attributes: [
             {
-                name: "focusHighlight",
-                description: "Specifies if button is highlighted when focused",
-                propTypes: ["boolean"],
-                defaultValue: "true",
+                name: "",
+                description: (
+                    <>
+                        This component also inherits props from&nbsp;
+                        <a
+                            href="https://developer.mozilla.org/en-US/docs/Web/API/HTMLButtonElement"
+                            rel="noreferrer"
+                            target="_blank"
+                        >
+                            HTMLButtonElement
+                        </a>
+                    </>
+                ),
             },
             {
-                name: "focusOutline",
-                description:
-                    "Controls the outline style when button is focused",
-                propTypes: [`"none"`, `"browser"`],
-                defaultValue: `"none"`,
+                name: "data-testid",
+                description: "The test identifier of the component",
+                propTypes: ["string"],
+            },
+
+            {
+                name: "buttonStyle",
+                description: "The navigation action button",
+                propTypes: [
+                    `"primary"`,
+                    `"secondary"`,
+                    `"light"`,
+                    `"disabled"`,
+                ],
+                defaultValue: `"primary"`,
```

Comment:
update the props accordingly

the description should be "The style of the button"